README.md

```
@@ -99,3 +127,29 @@ Join our community of developers creating universal apps.
 | weight_gram | numeric     | optional                             |
 | color       | text        | optional                             |
 | image_url   | text        | optional image                       |
+
+<h2  style="text-align: center;">Authors</h2>
+<div style="display: flex; flex-wrap: wrap; gap: 40px; justify-content: center;">
+
+  <div style="text-align: center;">
+    <a href="https://github.com/EmpyreanMist">
+      <img src="https://avatars.githubusercontent.com/u/175473868?v=4" width="80" style="border-radius:50%;" />
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)